src/test/java/com/amazon/ion/conformance/ConformanceTestRunner.kt

```diff
@@ -15,17 +15,19 @@ object DefaultReaderConformanceTests : ConformanceTestRunner(
 object IncrementalReaderConformanceTests : ConformanceTestRunner(
     IonReaderBuilder.standard()
         .withCatalog(ION_CONFORMANCE_TEST_CATALOG)
-        .withIncrementalReadingEnabled(true)
+        .withIncrementalReadingEnabled(true),
+    additionalSkipFilter = { _, testName -> "Incomplete floats signal an error for unexpected EOF" in testName }
 )
 
 abstract class ConformanceTestRunner(
     readerBuilder: IonReaderBuilder,
-    additionalSkipFilter: (File, String) -> Boolean = { _, _ -> true }
+    /** A predicate that returns `true` iff the test case should be skipped. */
+    additionalSkipFilter: (File, String) -> Boolean = { _, _ -> false }
```

Comment:
🗺️ I changed this because it was confusing when I went to use it for the incremental reader conformance tests.